build.gradle

```diff
@@ -7,12 +7,50 @@ description = 'RxJava: Reactive Extensions for the JVM – a library for composi
 
 apply plugin: 'rxjava-project'
 apply plugin: 'java'
+apply plugin: 'findbugs'
+apply plugin: 'jacoco'
 
 dependencies {
     testCompile 'junit:junit-dep:4.10'
     testCompile 'org.mockito:mockito-core:1.8.5'
 }
 
+////////////////////////////////////////////////////////////////////
+// to run findbugs: 
+//     ./gradlew findbugsMain
+// then open build/reports/findbugs/main.html
+////////////////////////////////////////////////////////////////////
+
+findbugs {
+  ignoreFailures = true
```

Comment:
Yep thanks, I can make that value false now that I have figured out how to exclude findbugs from default build.